config/src/config.rs

```diff
@@ -498,6 +505,33 @@ impl Default for MempoolConfig {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
+pub struct SafetyRulesConfig {
+    pub backend: SafetyRulesBackend,
+    // Required path for on disk storage
+    pub path: PathBuf,
```

Comment:
then this probably can be part of the OnDiskStorage? so the enum becomes `OnDiskStorage(PathBuf)`

Reply:
From a rust perspective that's great... not sure how that would serialize well

Reply:
it works nicely